src/ControlsPanel.cpp

```
@@ -14,19 +14,39 @@
 
 namespace {
 
+// TEMP: write the drag/resize trace straight to a file, sidestepping wx's
+// logging framework (whose output depends on OpenCPN's own log-level
+// setting) entirely. Neither $HOME nor GetUserDataDir() were writable
+// inside the flatpak sandbox under test; GetPluginDataDir() is the
+// directory OpenCPN itself already created and manages for this plugin
+// (confirmed to exist from "using data dir: ..." in opencpn.log). To be
+// reverted before merge.
+void DiagLog(const wxString& msg) {
+  wxFile f(GetPluginDataDir("mayara_pi") + "/mayara-drag-trace.log",
```

Clang (14.0.6)

[warning] variable 'f' is not initialized

(cppcoreguidelines-init-variables)